tests/acceptance/features/bootstrap/WebUIPersonalGeneralSettingsContext.php

```diff
@@ -243,6 +243,72 @@ public function aPasswordErrorMessageShouldBeDisplayedOnTheWebUIWithTheText(
 		);
 	}
 
+	/**
+	 * @When the user sets profile picture to :filename from the uploaded files using the webUI
+	 *
+	 * @param string $filename
+	 *
+	 * @return void
+	 */
+	public function theUserSetsProfilePictureToFromTheUploadedFiles($filename) {
+		$this->personalGeneralSettingsPage->setProfilePic($filename, $this->getSession());
+	}
+
+	/**
+	 * @Given the user has set profile picture to :filename from the uploaded files
+	 *
+	 * @param string $filename
+	 *
+	 * @return void
+	 */
+	public function theUserHasSetProfilePictureToFromTheUploadedFiles($filename) {
+		$this->theUserSetsProfilePictureToFromTheUploadedFiles($filename);
+		$this->thePreviewOfProfilePictureShouldBeShownInTheWebui("");
+	}
+
+	/**
+	 * @Then /^the preview of profile picture should (not|)\s?be shown in the webUI$/
+	 *
+	 * @param string $shouldOrNot
+	 *
+	 * @return void
+	 */
+	public function thePreviewOfProfilePictureShouldBeShownInTheWebui($shouldOrNot) {
+		$should = $shouldOrNot !== "not";
+		if ($should) {
```

Comment:
This should be straightforward to check. 
```suggestion
		if ($should !== 'not') {
```